app/controllers/contacts/new.js

```diff
@@ -0,0 +1,177 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency-decorators';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { A } from '@ember/array';
+import { isBlank } from '@ember/utils';
+
+export default class ContactsNewController extends Controller {
+  TITLES_ARRAY = ['Basis informatie', 'Ontvangers', 'Contactgegevens'];
+
+  @service currentSession;
+  @service store;
+
+  @tracked step = 0;
+  @tracked inputType = '';
```

Comment:
Waarom hier een lege string? Kan gewoon `@tracked inputType;` zijn. Value wordt gezet in `setupController()` in de route.